sql/6_1_0-to-7_0_0_upgrade.sql

```diff
@@ -320,190 +320,190 @@ ALTER TABLE api_token ADD COLUMN `context` TEXT COMMENT 'context values that cha
 CREATE TEMPORARY TABLE lang_updates_610 (notes char(15), twodigit_notes char(2));
 START TRANSACTION;
 INSERT INTO lang_updates_610 VALUES
```

Comment:
bunch of newlines removed here, can ignore them with:
https://github.com/openemr/openemr/pull/5185/files?w=1